src/commands/info.rs

```diff
@@ -0,0 +1,61 @@
+use std::env;
+
+use prettytable::{Table, Row, Cell};
+
+use crate::options::Options;
+use crate::platform;
+use crate::table;
+
+
+pub async fn info(_options: &Options)-> Result<(), anyhow::Error>
+{
+    let mut table = Table::new();
+
+    table.add_row(Row::new(vec![
+        Cell::new("Cache"),
+        Cell::new(&platform::cache_dir()?.display().to_string()),
+    ]));
+    table.add_row(Row::new(vec![
+        Cell::new("Config"),
+        Cell::new(&platform::config_dir()?.display().to_string()),
+    ]));
+    if let Ok(current_exe) = env::current_exe() {
+        table.add_row(Row::new(vec![
+            if current_exe == platform::binary_path()? {
+                Cell::new("CLI Binary")
+            } else {
+                Cell::new("Custom Binary")
+            },
+            Cell::new(&current_exe.display().to_string()),
+        ]));
+    }
+    let data_dir = platform::data_dir()?;
+    if data_dir.exists() {
+        table.add_row(Row::new(vec![
+            Cell::new("Data"),
+            Cell::new(&data_dir.display().to_string()),
+        ]));
+    }
+    if cfg!(target_os="linux") {
+        use crate::server::linux::unit_dir;
+
+        table.add_row(Row::new(vec![
+            Cell::new("Service"),
+            Cell::new(&unit_dir(false)?.display().to_string()),
+        ]));
+    } else if cfg!(target_os="macos") {
+        use crate::server::macos::plist_dir;
+
+        table.add_row(Row::new(vec![
+            Cell::new("Service"),
+            Cell::new(&plist_dir(false)?.display().to_string()),
+        ]));
+    }
+
+    table.set_format(*table::FORMAT);
+
+    println!("EdgeDB uses the following local directories or paths:");
```

Comment:
Just say "paths" and terminate directories in output with a directory separator to clarify.